rpcs3/Emu/Cell/Modules/cellMsgDialog.cpp

```diff
@@ -320,7 +320,14 @@ s32 cellMsgDialogProgressBarSetMsg(u32 progressBarIndex, vm::cptr<char> msgStrin
 		if (auto dlg2 = rsxthr->shell_get_current_dialog())
 		{
 			if (auto casted = dynamic_cast<rsx::overlays::message_dialog*>(dlg2))
+			{
+				if (!msgString)
```

Comment:
There is a parameter somewhere after this block, move it to before the GSRender check.

Reply:
I had been told that MsgDialogBase doesn't exist when the overlay is active

Reply:
Tyhe block from L341-343 has nothing to do with MsgDialogBase. Just move the check up before attempting to create the native interface.

Reply:
I figured that keeping the order of the errors was important assuming someone went out of their way to check the precedence, is it not?

Reply:
No. The other error (not_opened) means the system could not create the dialog and is impossible on rpcs3 currently.